packages/app/models/stack-order/stack-order.ts

```diff
@@ -16,6 +17,8 @@ export const calculateStackAveragePrice = (order: StackOrder) => {
 
   order.cowOrders.forEach((cowOrder) => {
     if (cowOrder.executedBuyAmount === "0") return;
+    // @ts-expect-error
+    if (cowOrder.status !== CowOrderStatus.FULFILLED) return;
```

Comment:
Our `CowOrderStatus` and CoW SDK `OrderStatus` have the same values, but TS tries to infer the types from two different enums and has collisions. IN THIS CASE, we can drop our `CowOrderStatus` and simply use the one from CoW SDK to fix this issue

```suggestion
    if (cowOrder.status !== OrderStatus.FULFILLED) return;
```

Reply:
Yeah, it should be that, that was my first thought. But using `OrderStatus` breaks compilation unexpectedly.
 
<img width="1039" alt="image" src="https://github.com/SwaprHQ/stackly-ui/assets/5664434/326a3257-8a67-4b53-8715-7fb6020d4915">

Reply:
Was able to use `OrderStatus` 🥳